cassettes/test_pages_create.yaml

```diff
@@ -0,0 +1,113 @@
+interactions:
+- request:
+    body: '{"parent": {"page_id": null}, "properties": {"title": [{"text": {"content":
+      "Test Page - 2022-11-13 11:58:44.392249"}}]}, "children": []}'
+    headers:
+      accept:
+      - '*/*'
+      accept-encoding:
+      - gzip, deflate
+      authorization:
+      - Bearer secret_UWDjvhuUx693qHExqJxQ9nu1KabHlyBJ78u3vUM8YDF
```

Comment:
You probably want to renew your secret now that it has leaked, and then change the code so that it doesn't appear anymore :)

Reply:
wooooops all those cassettes confused me 😖 luckily it was just an empty environment specifically for those tests. My bad!